<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TravelAgencyUI</class>
 <widget class="QMainWindow" name="TravelAgencyUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>594</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TravelAgencyUI</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="customerTable">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>310</y>
      <width>411</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableWidget" name="reiseTable">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>110</y>
      <width>331</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditCustomerId">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>20</y>
      <width>113</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditFirstName">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>50</y>
      <width>171</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditLastName">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>80</y>
      <width>171</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>30</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>ID:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>50</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Vorname:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Nachname:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>200</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Reise:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>32</x>
      <y>380</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Buchungen:</string>
    </property>
   </widget>
   <widget class="QWebEngineView" name="webViewMap">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>510</y>
      <width>411</width>
      <height>80</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>594</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuDatei">
    <property name="title">
     <string>Datei</string>
    </property>
    <addaction name="actionDateiOeffnen"/>
    <addaction name="actionSpeichern"/>
   </widget>
   <widget class="QMenu" name="menuEintragssuche">
    <property name="title">
     <string>Eintragssuche</string>
    </property>
    <addaction name="actionEintragssuche"/>
   </widget>
   <addaction name="menuDatei"/>
   <addaction name="menuEintragssuche"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionDateiOeffnen"/>
   <addaction name="actionSpeichern"/>
   <addaction name="actionEintragssuche"/>
  </widget>
  <action name="actionDateiOeffnen">
   <property name="icon">
    <iconset>
     <selectedoff>:/icons/icons/open.png</selectedoff>
    </iconset>
   </property>
   <property name="text">
    <string>Datei Ã¶ffnen</string>
   </property>
  </action>
  <action name="actionEintragssuche">
   <property name="icon">
    <iconset theme="system-search"/>
   </property>
   <property name="text">
    <string>Eintragssuche</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actionSpeichern">
   <property name="icon">
    <iconset theme="document-save"/>
   </property>
   <property name="text">
    <string>Speichern</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
